### PR TITLE
Feature/plugin detail collapsing toolbar

### PR DESCRIPTION
**Pull request by nbradbury**
Updates the plugin detail screen to use a collapsing toolbar rather than a fixed toolbar. 

plugin detail

### CODE REVIEW

**Comment by daniloercoli** [comment]
Wondering if the `image_gradient_scrim` item needs to use a different gradient image, maybe with a lighter grey?

If you look at the screenshot below there is the scrim that cut in half the plugin picture.
device-2018-01-25-114006

**Comment by nbradbury** [comment]
> Wondering if the image_gradient_scrim item needs to use a different gradient image, maybe with a lighter grey?

You're right - that is a little too dark. Addressed in 16e9470.

Here's what it used to look like:

before

And here's what it looks like after lightening:

after

I can't really go any lighter than that since it causes the white toolbar icons to blend in too much with lighter backgrounds.

File: WordPress/src/main/res/drawable/media_settings_gradient_scrim.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <gradient
         android:angle="90"
         android:centerY="75%"
-        android:endColor="@color/black_translucent_50"
+        android:endColor="#602e4453"
```
Comment (daniloercoli) [inline WordPress/src/main/res/drawable/media_settings_gradient_scrim.xml]: Nice, it's much better now. Just wondering where the new color come from? Isn't there a light grey already defined in the prj?

**Comment by nbradbury** [comment]
I didn't find a suitable pre-existing color value, but I'm open to
suggestions.

File: WordPress/src/main/res/drawable/media_settings_gradient_scrim.xml (hunk 1 above)
Comment (daniloercoli) [reply WordPress/src/main/res/drawable/media_settings_gradient_scrim.xml]: `grey_dark_translucent_50` seems very close to your proposed color.

Comment (nbradbury) [reply WordPress/src/main/res/drawable/media_settings_gradient_scrim.xml]: OK, that works for me. I made the change in 0ea5229.

**Comment by daniloercoli** [comment]
Cool! :shipit:

**Pull request merged**